src/test/java/com/github/wovnio/wovnjava/HtmlConverterTest.java

```diff
@@ -117,6 +117,16 @@ public void testRemoveForm__Sanitize__SingleQuotes() throws ConfigurationError {
         assertEquals(sanitized, stripExtraSpaces(converter.restore(html)));
     }
 
+    public void testRemoveForm__MultipleHiddenFields() throws ConfigurationError {
+        String original = "<HTML lang=\"en\"><HEAD></HEAD><BODY><FORM><INPUT type=\"hidden\" name=\"field_1\" value=\"text\"><INPUT type=\"hidden\" name=\"field_2\" value=\"\"><INPUT type=\"hidden\" name=\"field_3\" value=\"1\"><INPUT type=\"hidden\" name=\"field_4\" value=\"\"><INPUT type=\"hidden\" name=\"field_5\" value=\"\"><INPUT type=\"hidden\" name=\"field_6\" value=\"1234\"><INPUT type=\"hidden\" name=\"field_7\" value=\"000\"><INPUT type=\"hidden\" name=\"field_8\" value=\"1234-000\"><INPUT type=\"hidden\" name=\"field_9\" value=\"\"><INPUT type=\"hidden\" name=\"field_10\" value=\"\"><INPUT type=\"hidden\" name=\"field_11\" value=\"\"><INPUT type=\"hidden\" name=\"field_12\" value=\"0\"><INPUT type=\"hidden\" name=\"field_13\" value=\"\"><INPUT type=\"hidden\" name=\"field_14\" value=\"\"><INPUT type=\"hidden\" name=\"field_15\" value=\"\"><INPUT type=\"hidden\" name=\"field_16\" value=\"\"><INPUT type=\"hidden\" name=\"field_17\" value=\"0\"><INPUT type=\"hidden\" name=\"field_18\" value=\"\"><INPUT type=\"hidden\" name=\"field_19\" value=\"\"><INPUT type=\"hidden\" name=\"field_20\" value=\"\"><INPUT type=\"hidden\" name=\"field_21\" value=\"https://example.com/a\"><INPUT type=\"hidden\" name=\"field_22\" value=\"https://example.com/b\"><INPUT type=\"hidden\" name=\"field_23\" value=\"https://example.com/c\"><INPUT type=\"hidden\" name=\"field_24\" value=\"https://example.com/d\"><INPUT type=\"hidden\" name=\"field_25\" value=\"\"><INPUT type=\"hidden\" name=\"field_26\" value=\"\"><INPUT type=\"hidden\" name=\"field_27\" value=\"\"><INPUT type=\"hidden\" name=\"field_28\" value=\"foo\"><INPUT type=\"hidden\" name=\"field_29\" value=\"\"><INPUT type=\"hidden\" name=\"field_30\" value=\"\"><INPUT type=\"hidden\" name=\"field_31\" value=\"100\"></FORM></BODY></HTML>";
```

Comment:
Just do this with a loop to make this more readable
50 input fields, 50 elements with wovn-ignore on them

Reply:
Fixed in 89689d3daa44b557883bd3b23f4afb742aa2083c.

Adding `<div wovn-ignore>text</div>` makes the test pass for some reason. Though it fails if it's just the inputs.

Reply:
that's weird but OK we can leave just the inputs